docs/plugins/replaygain.rst

```diff
@@ -100,6 +116,15 @@ These options only work with the "command" backend:
   would keep clipping from occurring.
   Default: ``yes``.
 
+This option only works with the "bs1770gain" backend:
+
+- **method**:either replaygain, ebu or atsc. Default: replaygain
```

Comment:
Suggested text:

    - **method**: The loudness scanning standard; either `replaygain` for ReplayGain 2.0, `ebu` for EBU R128, or `asc` for ATSC A/85. This dictates the reference level: -18, -23, or -24 LUFS, respectively. Default: `replaygain`